src/commands/geosearchstore.json

```diff
@@ -75,12 +75,14 @@
                     {
                         "token": "FROMMEMBER",
                         "name": "member",
-                        "type": "string"
+                        "type": "string",
+                        "optional": true
```

Comment:
Try moving "optional" to the "oneof" container wrapping this one.

Reply:
Yes, that fixes this issue: https://github.com/valkey-io/valkey/pull/1809#discussion_r2096401973